src/vcml/debugging/gdbserver.cpp

```diff
@@ -789,6 +810,14 @@ void gdbserver::add_target(target* tgt) {
     if (!arch->collect_core_regs(*tgt, cpuregs))
         VCML_ERROR("target does not support %s", tgt->arch());
 
+    for (const auto& feature : arch->features) {
+        vector<const cpureg*> feature_cpuregs;
+        if (feature.collect_regs(*tgt, feature_cpuregs))
+            log_debug("gdb feature %s is supported", feature.name);
```

Comment:
Can we include the target name that supports/doesnt support a particular feature?